client/shipping-label/state/actions.js

```diff
@@ -265,8 +299,12 @@ export const purchaseLabel = () => ( dispatch, getState, { purchaseURL, addressN
 				console.error( error );
 				dispatch( NoticeActions.errorNotice( error.toString() ) );
 			} else {
-				// TODO: Figure out how to print multiple labels
-				printDocument( sprintf( labelImageURL, response[ 0 ].label_id ), nonce )
+				let pckgIndex = 1;
```

Comment:
You can avoid this counter by using the second argument passed to `Array.prototype.map` callbacks.